oneview_redfish_toolkit/tests/api/test_zone.py

```diff
@@ -31,29 +31,50 @@ class TestZone(BaseTest):
               'AvailableTargetsForSPT.json') as f:
         available_targets = json.load(f)
 
+    with open('oneview_redfish_toolkit/mockups/oneview/'
+              'Drives.json') as f:
+        drives = json.load(f)
+
     with open('oneview_redfish_toolkit/mockups/redfish/Zone.json') as f:
         zone_mockup = json.load(f)
 
-    def test_class_instantiation(self):
-        # Tests if class is correctly instantiated and validated
-        try:
-            zone = Zone(self.server_profile_template, self.available_targets)
-        except Exception as e:
-            self.fail("Failed to instantiate Zone class."
-                      " Error: {}".format(e))
-        self.assertIsInstance(zone, Zone)
+    with open('oneview_redfish_toolkit/mockups/redfish/'
+              'ZoneWithoutDrives.json') as f:
+        zone_without_drives_mockup = json.load(f)
 
     def test_serialize(self):
-        # Tests the serialize function result against known result
-        try:
-            zone = Zone(self.server_profile_template, self.available_targets)
-        except Exception as e:
-            self.fail("Failed to instantiate Zone class."
-                      " Error: {}".format(e))
-
-        try:
-            result = json.loads(zone.serialize())
-        except Exception as e:
-            self.fail("Failed to serialize. Error: {}".format(e))
+        zone = Zone(self.server_profile_template,
```

Comment:
Test doc is missing.

Reply:
Do you know is necessary for this case of tests?